### PR TITLE
feat(infra): VN-73: make a cyphernode server

### PR DESCRIPTION
**Pull request by gbibeaul**
**Description**
---
This adds a cloudformation template and associated runbook that can help deploy cyphernode on an ARM 64 ubuntu server on AWS. 

**Details**
---

- [x] Instance ressource with hard coded ami to ubuntu 18 arm server 
- [x]  security group ressource that opens port 22 for ssh access
- [x] runbook that explains how to replicate server and install docker on it 

**To do**
---
- [ ] get information on which port to open trough security group
- [ ] setup cyphernode on machine(git repo is cloned)

### CODE REVIEW

File: runbooks/deployCyphernodeAWS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Run book how to create an online cyphernode instance
+
+1. Generate a key pair in ec2 console and download it https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:
+1. connect to cloud formation https://console.aws.amazon.com/cloudformation/home?region=us-east-1
```
Comment (drawm) [inline runbooks/deployCyphernodeAWS.md]: Can you add more detailed steps on how to do this for those of us who never used couldformation?

Looking at the page at #2 the next step could look like : 
```
* In the `Prerequisite`, select `Template is ready`
* In the `Specify template`, select `Upload a template file`
```

File: runbooks/deployCyphernodeAWS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+1. deploy new stack using template file `infra\cyphernode-ec2-template.json`
+1. grab from the outputs section after the deployment is complete the public dns
+1. go from the console on a unix system where the console is located and run (replace with appropriate key name) `chmod 400 guillaume.pem && ssh -i "guillaume.pem" ubuntu@ec2-54-198-99-73.compute-1.amazonaws.com`
+1. generate an ssh key and add it to deploy keys in the git hub org.
```
Comment (drawm) [inline runbooks/deployCyphernodeAWS.md]: Why is this step needed? (why is github involved?)

Comment (gbibeaul) [reply runbooks/deployCyphernodeAWS.md]: To authenticate to github when we need to pull the code. Won't be needed when repo is public or if we store artifacts of the repo elsewhere (ex: s3)

File: runbooks/deployCyphernodeAWS.md (hunk 1 above)
Comment (gbibeaul) [reply runbooks/deployCyphernodeAWS.md]: updated!

**Pull request merged**